Sources/BigInt/BigInt.swift

```diff
@@ -1,21 +1,118 @@
-//===--- BigInt.swift ----------------------------------------*- swift -*-===//
+//===--- BigInt.swift -----------------------------------------*- swift -*-===//
 //
 // This source file is part of the Swift Numerics open source project
 //
-// Copyright (c) 2019 Apple Inc. and the Swift Numerics project authors
+// Copyright (c) 2019 - 2020 Apple Inc. and the Swift Numerics project authors
 // Licensed under Apache License v2.0 with Runtime Library Exception
 //
 // See https://swift.org/LICENSE.txt for license information
 //
 //===----------------------------------------------------------------------===//
 
-public struct BigInt: SignedInteger, LosslessStringConvertible {
-  public private(set) var words: [UInt]
+public struct BigInt: SignedInteger {
+
+  public typealias Words = [UInt]
```

Comment:
So this is strictly unnecessary as Swift will infer it based on the type of the `words` property, but just like with the `hash(into:)` I suppose it doesn't hurt to be explicit!